test/kubemark/resources/heapster_template.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Basically, my fear is that we will be lagging with the version, because noone will ever remember about changing this file.

Also, it seems that in our production we are using deployment :)